docs/en/observability/metrics-threshold-alert.asciidoc

```diff
@@ -11,13 +11,26 @@ conditions that combine metrics and thresholds to create precise notifications.
 . In the side navigation, expand *Observability*, click *Metrics*, and then click *Metrics Explorer*.
 . On the *Metrics Explorer* page, click *Alerts*, and then select *Create alert*.
 
+[TIP]
+=====
+You can create an alert from every graph displayed on the *Metrics Explorer* page. Click *Actions* and 
+then select *Create alert*. The condition, filter, and the create per sections of the threshold alert
```

Comment:
I can't quite parse this phrase (maybe because I'm not looking at the UI): "and the create per sections of the threshold alert"

TBH, I would avoid describing specific fields that get populated automatically because that might change in the future. Users will see that the fields are populated when they use the UI anyhow.